dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -72,24 +72,24 @@ BEGIN
     POPUP ""
     BEGIN
         MENUITEM SEPARATOR
-        MENUITEM "&Decupează", IDM_CUT
-        MENUITEM "&Copiază", IDM_COPY
-        MENUITEM "&Lipește", IDM_INSERT
+        MENUITEM "&Decupare", IDM_CUT
+        MENUITEM "&Copiere", IDM_COPY
+        MENUITEM "&Lipire", IDM_INSERT
         MENUITEM SEPARATOR
-        MENUITEM "Crează sc&urtătură", IDM_CREATELINK
-        MENUITEM "Șt&erge", IDM_DELETE
-        MENUITEM "&Redenumește", IDM_RENAME
+        MENUITEM "Creare c&omandă rapidă", IDM_CREATELINK
+        MENUITEM "Ş&tergere", IDM_DELETE
+        MENUITEM "Redenu&mire", IDM_RENAME
         MENUITEM SEPARATOR
-        MENUITEM "&Proprietăți", IDM_PROPERTIES
+        MENUITEM "Prop&rietăţi", IDM_PROPERTIES
     END
 END
 
 IDM_DRAGFILE MENU
 BEGIN
     POPUP ""
     BEGIN
-        MENUITEM "&Copiază aici", IDM_COPYHERE
-        MENUITEM "&Mută aici", IDM_MOVEHERE
+        MENUITEM "&Copiere în acest loc", IDM_COPYHERE
+        MENUITEM "&Mutare în acest loc", IDM_MOVEHERE
         MENUITEM "Crea&ză scurtătură aici", IDM_LINKHERE
         MENUITEM SEPARATOR
         MENUITEM "Revocare", 0
```

Comment:
Nitpick: At line 55 of this file you called this action "Anulare", but in this menu it is still called Revocare. Shall we use Anulare here as well? (You can reach this menu by dragging a file with the left mouse-button from one folder to another in Windows Explorer in MS Windows).